doc/smart-switch/pmon/static-ip-assignment.md

```diff
@@ -0,0 +1,51 @@
+# Midplane Interface 
```

Comment:
@rameshraghupathy can we remove this since smartswitch advocates for DHCP method for IP assignment?

Reply:
Done